scrypt.js

```diff
@@ -1,4 +1,5 @@
 document.body.innerHTML = '<h1>hello js</h1>';
 console.log('test1')
 console.log('test2!!!!!!!!!!')
-console.log('test2!!!!!!!!!!')
\ No newline at end of file
+console.log('test2!!!!!!!!!!')
```

Comment:
This line is duplicated with the previous one, maybe let's put a different message here?